CHANGELOG

```diff
@@ -6036,6 +6036,7 @@ devel
 
 * Fixed an agency bug found in Windows tests.
 
+* Limited module resolution in arangosh to the path from which arangosh is invoked.
```

Comment:
Please move this entry to the top of the CHANGELOG, so it becomes the "newest" entry in the "devel" section.